test_data/provider_defs/pangaea.json

```diff
@@ -0,0 +1,6 @@
+{
+  "type": "PangaeaDataProviderDefinition",
+  "id": "e3b93bf3-1bc1-48db-80e8-97cfb0685e8d",
```

Comment:
We could also think if we make it single providers.
You can take a look into the NetCdfCf provider and decide if this would make sense to get rid of the dual of setting it here and in the config to make it work or not.

Reply:
b1481f25b7c16ed9cb62d88aa207f3c69c31361b